module_utils/nxos/config/base.py

```diff
@@ -0,0 +1,16 @@
+from ansible.module_utils.connection import Connection
+
+
+class ConfigBase(object):
+
+    _connection = None
+
+    def __init__(self, module):
+        self._module = module
+        self._connection = self._get_connection()
+
+    def _get_connection(self):
```

Comment:
This API can be moved to the base class?

Reply:
@ganeshrn which base class are you referring here?